tests/test_webservice.py

```diff
@@ -215,16 +211,18 @@ def test_200_on_request_with_nones(self):
         response = self.response
         self.assertEqual(0.01, model.risk_score)
 
-    @httprettified
     def test_200_with_email_hashing(self):
-        uri = "/".join([self.base_uri, self.type])
+        uri = "/".join(["/minfraud/v2.0", self.type])
 
-        httpretty.register_uri(
-            httpretty.POST,
-            uri=uri,
-            status=200,
-            body=self.response,
-            content_type=f"application/vnd.maxmind.com-minfraud-{self.type}+json; charset=UTF-8; version=2.0",
+        last = None
+
+        def custom_handler(r):
```

Comment:
I think this should be updated similar to how the `geoip2` one was updated.

Reply:
I wrote a comment before about why this is still using custom_handler. https://github.com/maxmind/minfraud-api-python/pull/148#discussion_r1669332847
This specific test is checking the data of the request **not the headers** .  I found this doc about how to do that RequestMatcherhttps://pytest-httpserver.readthedocs.io/en/latest/howto.html#querying-the-log
However the class used for doing this "RequestMatcher" is not available on the release 1.0.10 (which is the latest release I installed from pip) but from master.

Reply:
Ah, I am sorry, I didn't realize that comment was referring to that. Is there a reason why `self.httpserver.expect_request(uri, method="POST", json=request)...` doesn't work?

Reply:
I think it can work but we have to modify the assertion. How do you suggest to do that?

Reply:
I am not sure I fully understand. I would expect something like:

```python
    def test_200_with_email_hashing(self):
        uri = "/".join(["/minfraud/v2.0", self.type])

        self.httpserver.expect_request(
            uri, 
            method="POST",
            json={
                "email": {
                    "address": "977577b140bfb7c516e4746204fbdb01",
                    "domain": "maxmind.com",
                }
            },
        ).respond_with_json(self.response)
```

Reply:
Right, my suggested code moved that to the `expected_request`. If the request does not match that expectation, the server will return a 500 and the test will fail.